### PR TITLE
Remove outdated dependabot config section

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##
This PR removes an outdated section of the dependabot config and also adds the ignore directive for Terraform, since that is handled by `cisagov/skeleton-tf-module`.

## 💭 Motivation and context ##

I removed the `/examples/basic_usage` directory in PR #4, but neglected to remove the corresponding dependabot config section.  I also forgot to enable the dependabot ignore directive for Terraform.  This PR remedies those oversights.

## 🧪 Testing ##

Visual inspection.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -32,11 +32,3 @@ updates:
     # # Managed by cisagov/cool-iam-user-group-manager-iam
```
Comment (jsf9k) [inline .github/dependabot.yml]: Not sure what happened here:
```suggestion
    # # Managed by cisagov/skeleton-tf-module
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
May as well fix this other thing while you're here.

File: .github/dependabot.yml (hunk 1 above)
Comment (dav3r) [reply .github/dependabot.yml]: Yeah, not sure what happened there either, but I cleaned it up in https://github.com/cisagov/cool-iam-user-group-manager-iam/pull/7/commits/069b354cf451d35936103a35ddcaad9f76cf49fc.

**Review by mcdonnnj — APPROVED** [review]
Knew I was forgetting something in my review 😑

File: .github/dependabot.yml (hunk 1 above)
Comment (mcdonnnj) [reply .github/dependabot.yml]: @dav3r I fixed this in https://github.com/cisagov/gh-skeleton/pull/30.

**Review by jsf9k — APPROVED** [review]

**Pull request merged**